src/components/styles/PageSection.styled.ts

```diff
@@ -12,23 +12,42 @@ const fadeInUp = keyframes`
     }
 `
 
-export const MainContainer = styled.div<{ imgPosition?: 'left' | 'right' }>`
-  display: grid;
+export const MainContainer = styled.div<{ imgPosition?: 'left' | 'right'; responsiveReverse?: boolean }>`
+  display: flex;
   justify-content: space-between;
-  grid-template-rows: minmax(0, 1fr);
-  grid-template-columns: ${(props) => props.imgPosition && 'auto'} auto;
-  grid-template-areas: '${(props) => props.imgPosition === 'left' && 'image'} text ${(props) =>
-    props.imgPosition === 'right' && 'image'}';
-  gap: 5rem;
   animation-duration: 1s;
   animation-name: ${fadeInUp};
+
+  @media screen and (min-width: 700px) {
+    display: grid;
+    grid-template-rows: minmax(0, 1fr);
+    grid-template-columns: ${(props) => props.imgPosition && 'auto'} auto;
+    grid-template-areas: '${(props) => props.imgPosition === 'left' && 'image'} text ${(props) =>
+      props.imgPosition === 'right' && 'image'}';
+    gap: 5rem;
+  }
+
+  @media screen and (max-width: 700px) {
+    flex-direction: ${(props) => (props.responsiveReverse ? 'column-reverse' : 'column')};
+    align-items: flex-start;
+    padding: 0px;
+    margin-bottom: 50px;
+  }
 `
 
 export const ImageContainer = styled.img`
-  grid-area: image;
   object-fit: contain;
   max-height: 350px;
   max-width: 350px;
+
+  @media screen and (min-width: 700px) {
```

Comment:
i think can do without this media query? will it affect the phone view if it's left in and using max-width media query?

Reply:
Previously it did but could be due to my local host lagging, seems to not affect now that I try again, I will remove this then.